documentation/asciidoc/computers/camera/rpicam_options_common.adoc

```diff
@@ -44,533 +48,515 @@ Available cameras
+Sets the supplied string as the title of the preview window when running in a desktop environment. Supports the following image metadata substitutions:
 
 |===
 | Directive | Substitution
 
 | %frame
-| The sequence number of the frame
+| Sequence number of the frame.
 
 | %fps
-| The instantaneous frame rate
+| Instantaneous frame rate.
 
 | %exp
-| The shutter speed used to capture the image, in microseconds
+| Shutter speed used to capture the image, in microseconds.
 
 | %ag
-| The analogue gain applied to the image in the sensor
+| Analogue gain applied to the image in the sensor.
 
 | %dg
-| The digital gain applied to the image by the ISP
+| Digital gain applied to the image by the ISP.
 
 | %rg
-| The gain applied to the red component of each pixel
+| Gain applied to the red component of each pixel.
 
 | %bg
-| The gain applied to the blue component of each pixel
+| Gain applied to the blue component of each pixel.
 
 | %focus
-| The focus metric for the image, where a larger value implies a sharper image
+| Focus metric for the image, where a larger value implies a sharper image.
 
 | %lp
-| The current lens position in dioptres (1 / distance in metres).
+| Current lens position in dioptres (1 / distance in metres).
 
 | %afstate
-| The autofocus algorithm state (one of `idle`, `scanning`, `focused` or `failed`).
+| Autofocus algorithm state (`idle`, `scanning`, `focused` or `failed`).
 |===
 
-When not provided, the `--info-text` string defaults to `"#%frame (%fps fps) exp %exp ag %ag dg %dg"`.
+image::images/focus.jpg[Image showing focus measure]
 
-Example: `rpicam-hello --info-text "Focus measure: %focus"`
+==== `width` and `height`
 
-image::images/focus.jpg[Image showing focus measure]
+Each accepts a single number defining the dimensions, in pixels, of the captured image.
 
-==== Camera resolution and readout
+For `rpicam-still`, `rpicam-jpeg` and `rpicam-vid`, specifies output resolution.
 
-----
-	--width				Capture image width <width>
-	--height			Capture image height <height>
-----
+For `rpicam-raw`, specifies raw frame resolution. For cameras with a 2×2 binned readout mode, specifying a resolution equal to or smaller than the binned mode captures 2×2 binned raw frames.
+
+For `rpicam-hello`, has no effect.
+
+Examples:
+
+* `rpicam-vid -o test.h264 --width 1920 --height 1080` captures 1080p video.
+
+* `rpicam-still -r -o test.jpg --width 2028 --height 1520` captures a 2028×1520 resolution JPEG. If used with the HQ camera, uses 2×2 binned mode, so the raw file (`test.dng`) contains a 2028×1520 raw Bayer image.
+
+==== `viewfinder-width` and `viewfinder-height`
+
+Each accepts a single number defining the dimensions, in pixels, of the image sent to for display in the preview window. Does not effect the preview window, since images are resized to fit. Does not effect captured still image or videos.
```

Comment:
"sent to for display" ?

Reply:
effect -> affect ?

Reply:
```suggestion
Each accepts a single number defining the dimensions, in pixels, of the image displayed in the preview window. Does not effect the preview window dimensions, since images are resized to fit. Does not affect captured still images or videos.
```